pages/wordpress-proxy.js

```diff
@@ -46,6 +48,29 @@ class WordpressProxy extends Component {
 
     return (
       <React.Fragment>
+        <Head>
+          <link href="/static/slider-revolution/css/slider-revolution.css" rel="stylesheet" />
+
+          <script src="/static/slider-revolution/js/jquery.js" />
```

Comment:
😢

Reply:
I knoooooooooooooow. On the plus side, it's *only* on Wordpress pages.